vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -146,6 +145,12 @@
     // jump to a group details sheet
     public static final String EXTRA_GROUP_ID = "VectorHomeActivity.EXTRA_GROUP_ID";
 
+    // add an extra to precise the type of mode we want to open the VectorRoomCreationActivity
+    public static final String EXTRA_ROOM_CREATION_ACTIVITY_MODE = "VectorHomeActivity.EXTRA_ROOM_CREATION_ACTIVITY_MODE";
+    public enum RoomCreationModes {
+        DIRECT_CHAT, DISCUSSION;
+    }
+
```

Comment:
Can we define `EXTRA_ROOM_CREATION_ACTIVITY_MODE` in the VectorRoomCreationActivity?
Same remark about `RoomCreationModes` enum